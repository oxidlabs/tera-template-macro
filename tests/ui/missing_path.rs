use tera_template_macro::TeraTemplate;

#[derive(TeraTemplate, serde::Serialize)]
#[template(not_path = "hello world")]
struct Smoke {
    hello: String
}

fn main() {}