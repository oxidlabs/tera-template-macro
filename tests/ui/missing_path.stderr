error: Expected `path`.
 --> tests/ui/missing_path.rs:4:12
  |
4 | #[template(not_path = "hello world")]
  |            ^^^^^^^^
